src/Pages/Calculator/EndScreen.jsx

```diff
@@ -7,11 +7,15 @@ import { CVPListItem } from "../../ui-kit/ListItem";
 import { BodyType } from "../../data/calculatorPagesTypes";
 import ProgressBar from "../../Components/ProgressBar";
 import { ExternalLink } from "../../ui-kit/ExternalLink";
-import PropTypes from 'prop-types';
+import PropTypes from "prop-types";
 
 const EndScreen = props => {
     const classes = EndScreenStyles();
 
+    const questionHistory = props.questionHistory;
+
+    console.log(props.questionHistory);
```

Comment:
Let's remove any stray `console.logs` before merging into staging branch : ) This was the MVP for the Hackathon, so it's great to know that we have all of the data at this point in the user's journey through the calc.